pipelines/jobs/configurations/jdk11u_pipeline_config.groovy

```diff
@@ -80,7 +80,8 @@ class Config11 {
                         hotspot: 'xlc13&&aix710',
                         openj9:  'xlc13&&aix715'
                 ],
-                test                : 'default'
+                test                : 'default',
+                cleanWorkspaceAfterBuild: true
```

Comment:
Does this not default to true in build_base_file.groovy already?

Reply:
No, this is the "pipeline" config, in build_base_file.groovy it sets "CLEAN_WORKSPACE": true, which means the generated build "job" config has it as true, so if you were to submit a build job: https://ci.adoptopenjdk.net/view/Failing%20Builds/job/build-scripts/job/jobs/job/jdk16/job/jdk16-aix-ppc64-openj9/build?delay=0sec
But, pipelines have a default of "false", eg.when you submit https://ci.adoptopenjdk.net/view/Failing%20Builds/job/build-scripts/job/openjdk11-pipeline/build?delay=0sec
The pipeline generated default is here: https://github.com/AdoptOpenJDK/openjdk-build/blob/0b12dd8c8ad8a7153363bd965b5d5d37690bb3de/pipelines/jobs/pipeline_job_template.groovy#L74